problemsets-mat223/linearalbegra.tex

```diff
@@ -3894,8 +3999,43 @@ \section*{Task 2.3: Pat and Jamie}
 	\]
 	\begin{parts}
 		\item How does the row space of $B$ relate to the row space of $C$?
+			\begin{solution}
+				They are equal.
+
+				Row operations replace rows with linear combinations of rows.
+				Therefore, since $C$ is the matrix $B$ after the application of
+				some row operations, $\Row(C)\subseteq\Row(B)$.
+
+				Since row operations are all reversible, we also know that $B$ 
+				can be obtained from $C$ by applying row operations, and so 
+				$\Row(B)\subseteq\Row(C)$.
+
+				Therefore, the two spans are equal. 
+			\end{solution}
 		\item How does the null space of $B$ relate to the null space of $C$?
+			\begin{solution}
+				They are equal. 
+
+				A vector is in $\Null(B)$ or $\Null(C)$ if and only if it is 
+				orthogonal to all vectors in $\Row(B)$ or all vectors in $\Row(C)$,
+				respectively. But the $\Row(B)=\Row(C)$ by the previous part, and 
+				so their null spaces must also be equal. 
+			\end{solution}
 		\item Compute the null space of $B$.
+			\begin{solution}
+				We compute $\Null(C)$, since it equals $\Null(B)$ by the previous part. 
+
+				$\vec x=\mat{x\\y\\z}$ is in $\Null(C)$ if and only if
+				\[
+					C\vec x = \mat{x-z\\y+2z} = \vec 0.
+				\]
+				This is true if $x=z$ and $y=-\frac{z}{2}$, so the null space is
```

Comment:
Just say, `The complete solution to this matrix equation is`...

Reply:
Done. Also putting the answer at the beginning, as I neglected to do earlier.